### ISSUE TITLE
Display own ratings in proposals list

### ISSUE THREAD

**Issue by Lysoun** [bpetetot/conference-hall-beta#796]
Hello,

It would be great to display one's ratings. It could help the reviewer to know which submission they already rated and have a general idea of what ratings they gave, to maybe adjust if they are all too positive or negative.

**Comment by bpetetot** [comment]
Do you mean you want to see your own ratings in the proposals list ? 

FYI: If you want to see proposals you already rated or not rated, you can use the filter "Ratings":
![image](https://user-images.githubusercontent.com/516360/81281827-a3ee9600-905a-11ea-9ef8-7cb971b203e9.png)

**Comment by Lysoun** [comment]
Yes that's what I mean. I think seeing my overall ratings would help me to rate. 

For example:
* if I notice that I only gave high ratings to submissions that concern one specific topic, I may change them to be more fair towards others
* if I tend to give very high ratings all the time, I would try to be more critical because giving a 5 to everything won't help with the decisions

**Comment by bpetetot** [comment]
Yes it's definitively a good idea, it should not be hard to implement I think.
We just have to see how to differentiate your own rating and the global rating to avoid confusion when you see the whole list.
